packages/templates/index.ts

```diff
@@ -0,0 +1 @@
+export *  from './react-urql-graphql-codegen/src/components/str/listPageTemplate'
```

Comment:
this folder is about a "mono repo" - `index.ts` is not suitable here since it is meant to be a package